util/ci/indent_tab_preprocess.py

```diff
@@ -0,0 +1,144 @@
+#!/usr/bin/python3
+
+import sys
+import re
+
+# preprocess files for indent check
+
+# Code processing:
+#
+# Tabs are keeped
+# Spaces after tabs is ignored if indent is not smaller
+#
+# Example:
+#
+# <tab>a = 6;
+# <spc>b = 5;
+# <tab><spc>if (
+# <tab><tab><tab><spc><spc>) {
+# <tab><tab>c = 7;
+# <tab><spc>}
+#
+# Result:
+#
+# <tab>a = 6;
+# <spc>b = 5;
+# <tab>if (
+# <tab><tab><tab>) {
+# <tab><tab>c = 7;
+# <tab><spc>}
+
+# Multiline comments processing:
+#
+# It is expected that tabs indent from comment begining is
+# used as indent on begining of every commented line.
+#
+# Example:
+#
+# <tab>/*
+# <spc><spc>A
+# <tab><tab>B
+# <tab><spc>C
+# D
+# <tab><spc>*/
+# <tab><tab>/*
+# <spc><tab><spc>R
+#	<tab><spc><tab>*/
+#
+# Result:
+#
+# <tab>/*
+# <spc>A
+# <tab>B
+# <tab>C
+# D
+# <tab>*/
+# <tab><tab>/*
+# <spc><tab>*R
+#	<tab><spc>*/
+
+
+def main():
+	if len(sys.argv) != 5:
+		#print("Bad arguments")
+		#print(sys.argv)
+		exit()
+	# multiline comment begining and end from command line
+	mlc_begin = sys.argv[1]
+	mlc_end = sys.argv[2]
+	mlc_end_len = int(sys.argv[3])
```

Comment:
```suggestion
	mlc_end_len = len(mlc_end)
```

Reply:
It can't be done. The regex comment line string does not have the same length as the non-regex representation.

Reply:
Can you use `re.escape()`?